src/tests/MedStickerEncryptionCharlie.contract.test.js

```diff
@@ -0,0 +1,91 @@
+import {
+  encrypt,
+  decrypt,
+  hash,
+  splitKeys,
+  fingerprint
+} from "../lib/MedStickerEncryptionCharlie";
+import {
+  arrayBufferToHex,
+  arrayBufferToString,
+  stringToArrayBuffer
+} from "../lib/utilities";
+import create from "../lib/factory";
+import { CHARLIE_STATIC_SALT } from "../lib/constants";
+
+const gcm = create("AES-GCM");
+
+const hexToArrayBuffer = hex => {
+  if (typeof hex !== "string") {
+    throw new TypeError("Expected input to be a string");
+  }
+
+  if (hex.length % 2 !== 0) {
+    throw new RangeError("Expected string to be an even number of characters");
+  }
+
+  const view = new Uint8Array(hex.length / 2);
+
+  for (let i = 0; i < hex.length; i += 2) {
+    view[i / 2] = parseInt(hex.substring(i, i + 2), 16);
+  }
+
+  return view.buffer;
+};
+
+describe("test utilities ", () => {
+  it("should convert hex string to ArrayBuffer", async () => {
+    expect(hexToArrayBuffer("00010a10f0ff")).toEqual(
+      new Uint8Array([0, 1, 10, 16, 240, 255]).buffer
+    );
+  });
+});
+
+describe("MedStickerEncryption contract, version charlie", () => {
+  // these all are contract values given from the other sources to validate the encryption,
+  // therefore are shared between out tests
+  const ORIGINAL_STRING = "MyVeryImportantData";
+  const MOCK_PIN = "someRandomPin12345678";
```

Comment:
```suggestion
  const PIN = "someRandomPin12345678";
```

Reply:
I would like to keep the `MOCK` prefix for the test values since we are doing it in all the other projects. 
I understand that is the point here but since this is simply a matter of taste I would ask you to be so kind and treat it accordingly. 
The rest will be  `MOCK_ BACKEND_SECRET`, `MOCK_ BACKEND_SALT`, `MOCK_IV`